### DISCUSSION TITLE
Driver gets Ignored

### DISCUSSION THREAD

**Discussion by omegazone26** [OpenTabletDriver/OpenTabletDriver#953] [Q&A] [answered]
Hello readers 

i wil come right away to the problem
So i just downloaded the OpenTabletDriver and commpleted all steps before downloading and after downloading. The first time when i started the Driver it worked fine, the area was set and the pen buttons etc. were disabled perfect for Osu! Now comes the but in the whole thing. After restarting my PC the Driver wouldnt change anything. I think its because of Wiindows Ink. but how can i Prevent windows Ink from dominating my CTH-470 before the OpenTabletDriver does? as meintiont it worrked once. then also the debugger function proved it. now nothing happens when debugging (heres some video proof didnt work so well as planned)

https://user-images.githubusercontent.com/64732806/113518191-132f7b80-9585-11eb-9032-6c0a3dff2477.mp4

i hope somone can help me

Wish you a good day
omegaone

P.S if you want a better video proof i will do one

**Comment by InfinityGhost** [comment, marked as answer]
As far as I'm aware, Windows Ink on most if not all Wacom tablets is a result of leftovers from the Wacom drivers. If the tablet is detecting in the log, the fix would be to kill off the Wacom driver software and re-detect the tablet.

**Reply by omegazone26** [reply]
so but i deleted all divers i could find do you have an tipp how to completely delete everything?

**Reply by InfinityGhost** [reply]
Searching through device manager for any leftovers would be a good start, but be careful with what you remove.

**Reply by omegazone26** [reply]
wow thannks i thout i only nneed to uuse systemmanager thanks now it works i deleted tablet and pen driver and after replugin the open driver works
